_rules/form-field-accessible-name-is-descriptive-e0ca31.md

```diff
@@ -14,34 +14,42 @@ input_aspects:
   - DOM Tree
   - CSS Styling
 authors:
+  - Anne Thyme Nørregaard
   - Dagfinn Rømen
```

Comment:
Following the discussion in the ACT-Rules meeting Thursday 12.09.2019, please consider taking Geir Sindre and my name off this rule. The rule has been under major revision since we last authored it.